MdePkg/Library/StackCheckLib/X64/StackCheckFunctionsMsvc.nasm

```diff
@@ -0,0 +1,45 @@
+;------------------------------------------------------------------------------
+; X64/StackCheckFunctionsMsvc.nasm
+;------------------------------------------------------------------------------
+
+    DEFAULT REL
+    SECTION .text
+
+extern ASM_PFX(StackCheckFailure)
+extern ASM_PFX(__security_cookie)
+extern ASM_PFX(CpuDeadLoop)
+
+; Called when a buffer check fails. This functionality is dependent on MSVC
+; C runtime libraries and so is unsupported in UEFI.
+global ASM_PFX(__report_rangecheckfailure)
+ASM_PFX(__report_rangecheckfailure):
+    call ASM_PFX(CpuDeadLoop)
+    ret
+
+; The GS handler is for checking the stack cookie during SEH or
+; EH exceptions and is unsupported in UEFI.
+global ASM_PFX(__GSHandlerCheck)
+ASM_PFX(__GSHandlerCheck):
+    call ASM_PFX(CpuDeadLoop)

```

Comment:
Same for this? Is it expected to handle it differently with another CPU dead loop call?

Reply:
Updated to a jmp